Config/StackBuilder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+use Liip\ImagineBundle\Factory\Config\StackFactory;
+
+final class StackBuilder implements StackBuilderInterface
+{
+    /**
+     * @var StackFactory
+     */
+    private $filterSetFactory;
```

Comment:
please rename to stackFactory (applies to all `filterSet` mentions in this class)